ivy/functional/backends/numpy/sorting.py

```diff
@@ -73,3 +73,15 @@ def searchsorted(
     else:
         ret = np.searchsorted(x, v, side=side, sorter=sorter)
     return ret.astype(ret_dtype)
+
+
+def lexsort(
```

Comment:
Also, are you sure that you need to implement the whole thing again not in the experimental folder? That seems redundent and your frontend function can just call your ivy.experimental function.

Reply:
I am getting an error when I don't implement it, so I will keep this in the current situation and after resolving other things I will evaluate this again. Would this be okay for you?